boards/arc/qemu_arc/qemu_arc_em.yaml

```diff
@@ -11,3 +11,4 @@ testing:
   ignore_tags:
     - net
     - bluetooth
+vendor: qemu
```

Comment:
I guess we should have snps here as well (and for rest of the ARC QEMU boards)?

CC @ruuddw @abrodkin